### ISSUE TITLE
Directory of missing Markdown and Illustrative Work.

### ISSUE THREAD

**Issue by bobgeorgethe3rd** [freesewing/freesewing#1342]
## Please read before helping.
The following is a list of all the missing and needed to be worked on Markdown and Illustrative Work. Ultimately the goal would be to get these all done before the v3 update so it is complete in conjuction with the update but it is not a deal breaker if it does not. Please note that cover arts are done by one specific artist and are on the list mostly for show, if you have an idea on how to represent the pattern though we would like to hear it.

To help, First check if there is already an issue assigned to the task you wish to help with.
- If there is go there and see if you can help.
- If it does not create an issue for the task and comment with the task and #issue number below so the list can be updated. Note: Tag me if the list hasn't been updated after 24 hours. You may also need to assign the Designers/Coders to the issue to get insight on the pattern.
- If it is strikethroughed/ticked it has been completed and does not need further assistance.
- If it is a simple fix just create a pull request with the solution rather than an issue. i.e. adding missing markdown.

If you think anything needs to be added/removed from the list please leave a comment below and it will be changed.

*Note*: A custom cutting layout system is currently being worked on and will more than likely supersede the need for cutting diagrams so these are not a priority.

Definitoin of Tags
**[illustrative]**
Stuff like the cover arts that are more interpretive.
**[diagram]**
Diagrams and illustrations for instructions/cutting diagrams/technical drawings etc.
**[markdown]**
Text amd formatting on the website such as instructions and other documentation.
**[open graph]**
Optimsing for the information taken when posting a link in places such as Discord
**[website]**
Incorrect displays that cannot be fixed by markdown.
   
#### All or a lot of
 - [x] Add yardage to all need pages and correct plural and singulars.
 - [x] Now that the split has finalised many now have markdown pattern options updates required **[markdown]** 
 
#### Aaron
 - [ ] Aaron cutting diagrams **[diagram]**

#### Albert
- [x] Albert Fabric Options proof read **[markdown]**
- [x] Albert Instructions **[markdown]** #1082
- [ ] Albert instruction diagrams **[diagram]**
- [ ] Albert cutting diagrams **[diagram]**

#### Bee
- [x] Bee Cover Art **[illustrative]**
- [ ] Bee Cross Back Tie Variant Line Drawing? **[diagram]**
- [ ] Bee cutting diagrams **[diagram]**

#### Bella
- [ ] Bella Cover Art **[illustrative]** #1626
- [x] Bella Fabric Options suggest mock-up/test fabrics for the default block. **[markdown]** #1413 
- [x] Bella basic instructions on how to construct the default block **[markdown]** #1413 
- [x] Bella What you need for the default block **[markdown]** #1413 
- [x] Update Bella note on instuctions documentation about blocks to state that the following info is for the deault as is block **[markdown]** #1413 
- [ ] Bella cutting diagrams for default block **[diagram]** 
- [ ] Bella instruction diagrams for default block **[diagram]**

#### Benjamin
- [ ] Benjamin cutting diagrams **[diagram]**

#### Bent
- [ ] Bent Cover Art **[illustrative]** #1626
- [x] Bent cutting markdown for default block **[markdown]**  #1440
- [x] Bent fabric options for default block mock-up **[markdown]**  #1440
- [x] Bent instructions for default block and update block note **[markdown]**  #1440
- [x] Bent What you need for default block **[markdown]**  #1440
- [x] Bent about being a block on instruction pages like Bella **[markdown]**  #1440
- [ ] Bent cutting diagrams for default block **[diagram]**
- [ ] Bent instructions diagrams for default block **[diagram]**

#### Breanna
- [ ] Breanna Cover Art **[illustrative]** #1626
- [x] Breanna Pattern Options Markdown, Not all is missing but a lot is and some have bent/brian diagrams that need to be removed/changed **[markdown]** #1445
- [x] Breanna Fabric Options suggest mock-up/test fabrics for the default block. **[markdown]** #1445
- [x] Breanna basic instructions on how to construct the default block **[markdown]** #1445
- [x] Breanna What you need for the default block **[markdown]** #1445
- [x] Update Breanna note on instuctions documentation about blocks to state that the following info is for the deault as is block **[markdown]** #1445
- [ ] Breanna cutting diagrams for default block **[diagram]**
- [ ] Breanna instruction diagrams for default block **[diagram]**

#### Brian
- [ ] Brian Cover Art **[illustrative]** #1626
- [x] Brian cutting markdown for default block **[markdown]** #1474
- [x] Brian fabric options for default block mock-up **[markdown]** #1474
- [x] Brian instuctions for default block and update block note **[markdown]** #1474
- [x] Brian What you need for default block **[markdown]** #1474
- [x] Brian about being a block on instruction pages like Bella/ alter pre-existing **[markdown]** #1474
- [ ] Brian cutting diagrams for default block **[diagram]**
- [ ] Brian instructions diagrams for default block **[diagram]**

#### Bob
- [ ] Bob Cover Art **[illustrative]**
- [x] Bob yaml size **[markdown]** #2312
- [ ] Bob instructions for bias binding fixMe **[markdown]**
- [x] Bob Pattern options **[markdown]** #2521
- [ ] Bob Instruction diagrams **[diagram]**
- [ ] Bob line drawing **[diagram]**

#### Bruce
- [ ] Bruce cutting diagrams **[diagram]**

#### Carlita
- [ ] Carlita Cover Art **[illustrative]**
- [ ] Carlita Instructions markdown update/upgrade from preliminary instructions **[markdown]** 
- [x] Carlita Fabric Options markdown **[markdown]** #1816
- [x] Carlita Cutting markdown **[markdown]** #1816
- [x] Carlita What you need **[markdown]** #1816
- [x] Carlita missing options markdowns **[markdown]** #1816
- [ ] Carlita cutting diagrams **[diagram]**
- [ ] Carlita using Carlton diagrams for pattern options. **[diagrams]**
- [ ] Carlita Instructions diagrams **[diagram]**
- [x] Carlita using Brian Pattern Options **[website]**  #1774

#### Carlton
- [ ] Carlton Cover Art **[illustrative]**
- [ ] Carlton Instructions markdown **[markdown]** 
- [x] Carlton Fabric Options markdown **[markdown]** #1816
- [x] Carlton Cutting markdown **[markdown]** #1816
- [x] Carlton What you need **[markdown]** #1816
- [x] Carlton missing options markdowns **[markdown]** #1816
- [ ] Carlton cutting diagrams **[diagram]**
- [ ] Carlton Instructions diagrams **[diagram]**
- [x] Carlton inccorect pattern options. **[website]** #1774

#### Cathrin
- [ ] Cathrin Re-format Instructions to bring into standard

#### Charlie
- [ ] Charlie Cover Art? Currently uses what is Theo's **[illustrative]**
- [x] Charlie Waistband Curve markdown **[markdown]**
- [x] Charlie Cross Seam angle markdown **[markdown]** #1475
- [x] Charlie Crotch Seam angle markdown **[markdown]** #1475
- [ ] Charlie cutting diagrams **[diagram]**
- [ ] Charlie instructions diagrams **[diagram]**

#### Cornelius
- [x] Cornelius Cuff Style markdown **[markdown]** #1412 
- [x] Cornelius cutting markdown **[markdown]** #1437 
- [x] Cornelius fabric options **[markdown]** #1437
- [x] Cornelius instructions update from prelim markdown **[markdown]** #1437
- [x] Cornelius what you need markdown **[markdown]** #1437
- [ ] Cornelius cutting diagrams **[diagram]**
- [ ] Cornelius instructions diagrams **[diagram]**

#### Diana
- [x] Diana Cover Art **[illustrative]**
- [x] Diana instructions re-format #1725 
- [ ] Diana cuttting diagrams **[diagram]**
- [ ] Diana instructions diagrams **[diagram]**
- [x] Diana using incorrect pattern options **[website]** #1774

#### Florent
- [ ] Florent possibly create instruction diagrams instead of pictures **[diagram]**

#### Florence
- [ ] Florence cutting diagrams **[diagram]**

#### Hi
- [ ] Hi Cover Art **[illustrative]**
- [x] Hi some format changes and proof read **[markdown]** #2295
- [x] Hi information on eye and gills **[markdown]** #2318
- [x] Hi yml file **[markdown]**
- [ ] Hi Line Drawing **[diagram]**
- [ ] Hi instructions diagrams **[diagram]**
- [ ] Hi cutting diagrams **[diagram]**

#### Holmes
- [ ] Holmes Cover Art **[illustrative]**
- [x] Holmes pattern options Markdown **[markdown]** #1402 
- [ ] Holmes cutting diagrams **[diagram]**
- [ ] Holmes instructions diagrams **[diagram]**

#### Hortensia
- [ ] Hortensia Cover Art **[illustrative]**
- [x] Hortensia pattern options markdown **[markdown]** #1836
- [x] Hortensia cutting makrdown **[markdown]** #1836
- [x] Hortensia Fabric options markdown **[markdown]** #1836
- [x] Hortensia instructions markdown **[markdown]** #1836
- [x] Hortensia what you need markdown **[markdown]** #1836
- [ ] Hortensia cutting diagrams **[diagram]**
- [ ] Hortensia instructions diagrams **[diagram]**

#### Huey
- [x] Huey cutting markdown **[markdown]** #1552
- [x] Huey instructions markdown **[markdown]** #1552
- [x] Huey what you need markdown **[markdown]** #1552
- [ ] Huey cutting diagrams **[diagram]**
- [ ] Huey instructions diagrams **[diagram]**
- [x] Huey using incorrect pattern options **[website]** #1774

#### Hugo
- [ ] Hugo cutting diagrams **[diagram]**
- [ ] Hugo instructions diagrams **[diagram]**
- [x] Hugo using incorrect pattern options **[website]** #1774

#### Jaeger
- [ ] Finish Jaeger instructions markdown **[markdown]**
- [ ] Jaeger full cutting diagrams **[diagram]**
- [ ] Missing Jaeger instruction diagrams once the above is done **[diagram]**
- [x] Jaeger using incorrect pattern options **[website]** #1774

#### Lucy
- [ ] Lucy's Cover art  **[illustrative]**
- [x] Lucy What you need Documentation **[markdown]** #2629
- [x] Lucy Pattern Options Documentation **[markdown]** #2629
- [x] Lucy yml file **[markdown]** 
- [x] Lucy Line Drawing **[diagram]** #2629
- [ ] Lucy instructions diagrams **[diagram]**
- [ ] Lucy cutting diagrams **[diagram]**

#### Lunetius
- [ ] Lunetius Cover art **[illustrative]**
- [x] Lunetius Fabric Options **[markdown]**
- [x] Lunetius Pattern Options re-formating to remove title errors **[markdown]** #1449
- [x] Lunetius Line Drawing **[diagram]**
- [ ] Lunetius cutting diagrams **[diagram]**

#### Noble
- [ ] Noble Cover Art **[illustrative]** #1626
- [x] Noble yml file **[markdown]** 
- [x] Noble yaml type **[markdown]** #2312 
- [x] Noble dart position page missing **[markdown]** #2313
- [x] Noble frontArmholeCuravture yml **[markdown]** #2640
- [ ] Noble Line Drawing **[diagram]**
- [ ] Noble cutting diagrams for default block **[diagram]**
- [ ] Noble instructions diagrams for default block **[diagram]**

#### Octoplushy
- [ ] Octoplushy Cover Art **[illustrative]**
- [ ] octoplushy.yml fixme infos **[markdown]**
- [ ] Octoplushy missing Pattern option pages **[markdown]**
- [ ] Octoplushy Line Drawing **[diagram]**
- [ ] Octoplushy instructions diagrams **[diagram]**

#### Paco
- [ ] Paco Cover Art **[illustrative]**
- [ ] Paco cutting diagrams **[diagram]**
- [x] Paco Cross seam angle pattern option info #3794
- [x] Paco Crotch seam angle pattern option info #3794

#### Penelope
- [x] Penelope cutting markdown bullet points **[markdown]**  #1558
- [x] Penelope instructions markdown formatting **[markdown]**  #1558
- [x] Penelope update Fabric Options **[markdown]**  #1558 
- [x] Penelope update what you need **[markdown]** #1558
- [ ] Penelope instructions diagrams **[diagram]**
- [ ] Penelope cutting diagrams **[diagram]**

#### Sandy
- [x] Sandy cutting markdown **[markdown]** #1664
- [x] Sandy fabric options markdown **[markdown]** #1664
- [x] Sandy instructions markdown **[markdown]** #1664
- [x] Sandy what you need markdown **[markdown]** #1664
- [ ] Sandy cutting diagrams **[diagram]**
- [ ] Sandy instructions diagrams after above **[diagram]**

#### Shin
- [ ] Shin instructions diagrams **[diagram]**

#### Simon
- [x] Simon Yoke Height  Opitons **[markdown]** #1776
- [ ] Simon cutting diagrams **[diagram]**
- [x] #1187 
- [x]  Simon using incorrect pattern options **[website]** #1774

#### Simone
- [x] Simone adding bust darts **[markdown]** #1551 
- [x] Simone Front darts Pattern Opitons **[markdown]** #1776
- [x] Simone Contour Pattern Opitons **[markdown]** #1776 
- [x] Simone Yoke Height  Opitons **[markdown]** #1776 
- [x] Simone Bust dart angle Options **[markdown]** #1776 
- [x] Simone Bust dart length Options **[markdown]** #1776 
- [x] Simone Front dart length Options **[markdown]** #1776 
- [ ] Simone Cover Art separate from Simons **[illustrative]**
- [ ] Simone cutting diagrams **[diagram]**
- [x] #1187
- [x] Simone using incorrect pattern options **[website]** #1774

#### Sven
- [ ] Sven cutting diagrams **[diagram]**
- [ ] Sven instructions diagrams **[diagram]**
- [x] Sven using incorrect pattern options **[website]** #1774

#### Tamkio
- [x] Tamkio instructions made need to be updated a little **[markdown]** #1667
- [ ] Tamkio cutting diagrams **[diagram]**

#### Teagan
- [ ] Teagan line drawing not representing the neckband **[diagram]**
- [ ] Teagan cutting diagrams **[diagram]**
- [x]  Teagan using incorrect pattern options **[website]** #1774

#### Theo
- [ ] Theo cutting diagram **[diagram]**
Note: Theo is depreceated and may be slated for overhaul at some point.

#### Tiberius
- [ ] Tiberius Cover art **[illustrative]**
- [x] Tiberius Pattern Options re-formating to remove title errors **[markdown]** #1449
- [x] Tiberius Line Drawing **[diagram]**
- [ ] Tiberius cutting diagrams **[diagram]**

#### Titan
- [ ] Titan Cover Art **[illustrative]**  #1626
- [x] Titan Re-fromatting to bring in line with the other blocks #1577
- [x] Titan about being a block on instrutction pages like Bella **[markdown]** #1577
- [x] Titan Cross seam angle pattern option info #3794
- [x] Titan Crotch seam angle pattern option info #3794
- [ ] Titan cutting diagrams for default block **[diagram]**
- [ ] Titan instructions diagrams for default block **[diagram]**

#### Unice
- [ ] Unice Cover Art **[illustrative]**
- [ ] Unice Line Drawing **[diagram]**
- [ ] Unice cutting diagrams **[diagram]**

#### Ursula
- [ ] Ursula cutting diagrams **[diagram]**

#### Trayvon
- [ ] Trayvon Re-format of Instructions to bring into standard

#### Wahid
- [ ] Wahid cutting diagrams **[diagram]**

#### Walburga
- [ ] Walburga Cover art **[illustrative]**
- [x] Walburga fabric options markdown **[markdown]**
- [x] Walburga Pattern Options re-formating to remove title errors **[markdown]** #1449
- [x] Walburga Line Drawing **[diagram]**
- [ ] Walburga cutting diagrams **[diagram]**

#### Waralee
- [x] Waralee Cover Art **[illustrative]**
- [x] Waralee Documentation re-format **[markdown]** #1813 
- [x] Waralee Pocket Instruction Update **[markdown]** #1575 
- [ ] Waralee cutting diagrams **[diagram]**

#### Yuri
- [ ] Yuri Cover Art **[illustrative]**
- [x] Yuri line drawing. **[diagram]**
- [ ] Yuri cutting diagram **[diagram]**
- [ ] Yuri instructions diagrams **[diagram]**
- [x] Yuri using incorrect pattern options **[website]** #1774

Once again if anything is missing or you believe needs changing comment below.

**Comment by starfetch** [comment]
Maybe this is also a good time to tackle issue #644 (the oldest open issue ;)).

**Comment by bobgeorgethe3rd** [comment]
Removed Open Graph Work.
Added
- Bee 
- Lunetius
- Tiberius
- Walburga

**Comment by bobgeorgethe3rd** [comment]
Re-organized bullet points so that the catergories are together.

Illustrative
Markdown
Diagram

**Comment by bobgeorgethe3rd** [comment]
Now all 41 patterns are on the labels :)

**Comment by bobgeorgethe3rd** [comment]
Noted. Editing the comment can sometimes undo the ticks

**Comment by bobgeorgethe3rd** [comment]
Added Website tag for those that cannot be fixed via markdown and re-updated diagrams that were actually incorrect pattern options pages to website tag

**Comment by bobgeorgethe3rd** [comment]
Updated to include, Lucy, Hi and Noble.

**Comment by bobgeorgethe3rd** [comment]
Updated to add Bob and Unice.

**Comment by joostdecock** [comment]
This issue is over 2 years old, and very stale. It's better to work in bite-sized chunks.
Closing, but great effort :+1:
